src/main/java/com/brandon3055/draconicevolution/client/render/item/RenderModularBow.java

```diff
@@ -145,7 +146,7 @@ public void renderTool(CCRenderState ccrs, ItemStack stack, TransformType transf
         }
 
         Matrix4 effectMat = mat.copy();
-        drawStrings(ccrs, mat, bottomMat, getter, drawAngle, packedLight);
+        drawStrings(ccrs, mat, bottomMat, getter, drawAngle, packedLight, getEnergyStored(stack) > 0);
```

Comment:
You need to check if the energy stored is greater or equal to the energy required to take a shot. 
Use `ModularBow.calculateShotEnergy(stack)`